### PR TITLE
stats for pli and bitrate

### PR DESCRIPTION
**Pull request by cnderrauber**

### CODE REVIEW

File: pkg/sfu/receiver.go [hunk 1]
```diff
@@ -402,6 +403,18 @@ func (w *WebRTCReceiver) SendPLI(layer int32) {
 	buff.SendPLI()
 }
 
+func (w *WebRTCReceiver) LastPLI() int64 {
+	var lastPLI int64
+	w.bufferMu.RLock()
+	for _, b := range w.buffers {
```
Comment (boks1971) [inline pkg/sfu/receiver.go:409]: Probably have to check `if b != nil`

File: pkg/sfu/downtrack.go [hunk 2]
```diff
@@ -483,6 +490,26 @@ func (d *DownTrack) WritePaddingRTP(bytesToSend int) int {
...
+	lastRtp := d.lastRTP.Load()
+	d.statsLock.RLock()
+	update := lastRtp.Sub(d.lastBitrateReport) > time.Second
+	d.statsLock.RUnlock()
+	if !update {
+		return
+	}
+	octets, _ := d.getSRStats()
+	d.statsLock.Lock()
+	d.bitrate = octets*8 - d.bitrateHelper
```
Comment (boks1971) [inline pkg/sfu/downtrack.go:503]: This might be okay, but to be accurate, this should take the time difference and scale like `sfu.buffer` does.

**Review by boks1971 — APPROVED** [review]
@cnderrauber A couple of comments.

On a broader node, am wondering if we should standardise on `RTPStats` module (would need to be moved to OSS) for everything. I know I did `StreamStats` module for that purpose. But, `RTPStats` module is contained in one place. Also, standardise on that data for all track stats?

**Comment by cnderrauber** [comment]
@boks1971 Agree for standardise RTPStats, actually  I move  RTPStats to OSS at first, but found have some duplicate to StreamStats, so I removed it :).

**Pull request merged**